shared/util/feature-flags.js

```diff
@@ -0,0 +1,11 @@
+/* @flow */
+
+import {trackerVersionTwo} from '../local-debug'
```

Comment:
sorry for the PR churn, i think its simpler to just have one or the other, either process.env or local-debug, i don't think we gain much by having both. so i'd just keep this file and just do 

```
tracker: process.env. KEYBASE_TRACKER_V2 ? 'v2' : 'v1'
```